sensor_msgs/Image rgb_image
sensor_msgs/Image depth_image
sensor_msgs/CameraInfo camera_info
std_msgs/String frame_id
std_msgs/Time timestamp
---
sensor_msgs/Image mask
std_msgs/Int16 centers
